### PR TITLE
Jquery overlays

### PR DESCRIPTION
**Pull request by leahgoldberg**

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -14,5 +14,14 @@
 //= require jquery_ujs
 //= require foundation
 //= require_tree .
+//= require jquery.leanModal.min.js
+//= require chosen.jquery.min.js
+//= require chosen.proto.min.js
+<<<<<<< HEAD
```
Comment (kripple) [inline app/assets/javascripts/application.js]: remove merge-y stuff

Comment (leahgoldberg) [inline app/assets/javascripts/application.js]: yeah, it's not in the actual file - not sure why it's showing up here.

File: app/models/user.rb [hunk 2]
```diff
@@ -39,6 +39,10 @@ def find_associated_coffees
 	  received_coffees << coffee_gift if coffee_gift
 	end
 
+	def full_name
+		[first_name, last_name].join(" ")
```
Comment (kripple) [inline app/models/user.rb:43]: why not `"#{first_name} #{last_name}"` ?

File: app/views/coffee_gifts/confirm.html.erb [hunk 3]
```diff
@@ -6,7 +6,7 @@
   <h2>Here are the details of your purchase: </h2>
 
   <% if @coffee_gift.receiver %>
-    <p>Sent To: <%=@coffee_gift.receiver.username%></p>
+    <p>Sent To: <%=@coffee_gift.receiver.full_name%></p>
```
Comment (kripple) [inline app/views/coffee_gifts/confirm.html.erb:9]: :+1:

**Pull request merged**